services/configuration/CHANGELOG.md

```diff
@@ -1,16 +1,53 @@
 # Changelog
 
+<!-- cspell:ignore ZEROVOTE -->
```

Comment:
What do you think about ignoring it locally? `ZEROVOTE` is removed, so we shouldn't encounter it in other places.